artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/etc/amqp-acceptor.txt

```diff
@@ -1,3 +1,3 @@
 
          <!-- AMQP Acceptor.  Listens on default AMQP port for AMQP traffic.-->
-         <acceptor name="amqp">tcp://${host}:${amqp.port}?protocols=AMQP;useEpoll=true</acceptor>
+         <acceptor name="amqp">tcp://${host}:${amqp.port}?protocols=AMQP;useEpoll=true;amqpCredits=1000;amqpMinCredits=300</acceptor>
```

Comment:
This configures 'amqpMinCredits', whereas broker.xml configures 'amqpLowCredits', is one of them wrong? If not, perhaps it would be clearer if they were the same?

It might also be clearer to rationalise the naming in the code as well, it seems to use 'amqpLowMark', 'amqpLowCredits', and 'minCreditRefresh' to refer to this.